admin-ui/src/styles/util.ts

```diff
@@ -168,12 +168,7 @@ export const Divider = styled.hr`
   grid-column: 1/-1;
 `;
 
-export const PlainButton = styled(Button).attrs({
-  variant: "secondary",
-  style: {
-    "--color-bus": "var(--color-black)",
-  },
-})``;
+export const PlainButton = styled(Button).attrs({ variant: "secondary" })``;
```

Comment:
Should the styles be applied?

Reply:
Don't exist in the new version of HDS-REACT. what was that styling? is it the background-color?

Reply:
In this case it overrides the blue color used in the component -- actually the css variable used inside the compoent. Not sure if this is the latest version, but still works here: https://hds.hel.fi/storybook/react/?path=/docs/components-button--full-width

Reply:
On further inspection, this is not even used. so it could be removed